mne/utils/docs.py

```diff
@@ -2499,13 +2499,36 @@
     If it is a 2-tuple of minimum and maximum limit, then those
     physical ranges will be used. Only used for exporting EDF files.
 """
+docdict['export_params_add_ch_type'] = """
+add_ch_type : bool
+    Whether to add the channel type of the channel currently set
+    (default is False) to the signal label. Only used for
+    exporting EDF files.
+"""
 docdict['export_eeglab_note'] = """
 For EEGLAB exports, channel locations are expanded to full EEGLAB format.
 For more details see :func:`eeglabio.utils.cart_to_eeglab`.
 """
 docdict['export_edf_note'] = """
-For EDF exports, only EEG, ECoG and sEEG data are supported. In
-addition, EDF does not support storing a montage.
+For EDF exports, only channels that are measured in terms of Voltage are
+supported. For example, 'eeg', 'ecog', 'seeg', 'bio', 'emg', 'eog',
+'ecg', 'dbs', and 'bio' data are supported. 'misc' channels are also
+allowed for a catch-all of "unknown" channel types. Channel types are able
+to be stored in the prefix of the signal label. For example, ``EEG Fz`` implies
+that ``Fz`` is an EEG channel and ``MISC E`` would imply ``E`` is a MISC
+channel. However, EDF files do not require the storage of channel types,
+so many writers do not export the channel type. This is because
+there is no standard way for specifying all channel types.
+
+If you want to set the channel types, then set the ``add_ch_type`` keyword.
+This will then for example take EEG channel ``Fz`` and export the
+signal label as ``EEG Fz`` in the EDF file. In order to properly set
+the channel types before exporting, you will need to call
+:attr:`raw.set_channel_types <mne.io.Raw.set_channel_types>`.
```

Comment:
this paragraph is mostly redundant with the previous paragraph and the docstring of `add_ch_type`.  The only non-redundant part is the last sentence, which I'm not sure is even accurate?  It implies that in (many? most? all?) cases, the channel types will not be properly set, and users will need to do something extra to get them properly set.  Can you clarify in which cases you think users will want/need to use `raw.set_channel_types()`?

Reply:
I updated the doc string here now. Lmk wyt.